.github/workflows/perf.yml

```
@@ -16,6 +16,7 @@ jobs:
       MOC_VERSION: 0.10.4
       IC_WASM_VERSION: 0.7.0
       RUSTC_VERSION: 1.75.0
+      MOC_ARTIFACT: 1254821742
```

```suggestion
      MOC_ARTIFACT: 1254959535
```
Update to latest artifact that optimize clearing of all zero tags